README.md

```diff
@@ -16,58 +16,121 @@ Open-source software-defined EDA
 
 </div>
 
-## Get running
+---
+
+## About
+
+### What \[is faebryk\]
+
+faebryk is an open-source software-defined EDA (electronic design automation) tool.
```

Comment:
Normally this would be the other way around: software-defined electronic design automation (EDA) tool.